lib/actions/usbsdfuTargetActions.js

```diff
@@ -438,11 +455,15 @@ function operateDFU(inputDfuImages, inputDevice) {
             if (error.message) {
                 logger.error(`Reopen device failed: ${error.message || error}`);
 
-                // Notify the user to reset the device without Nordic Trigger Interface.
+                // Notify the user to reset the device without Nordic DFU Trigger Interface.
                 if (error.message.startsWith('Timeout while waiting')) {
-                    logger.error('It seems that the APP does not have a Nordic Trigger Interface.');
-                    logger.error('Please physically reset the device by clicking the reset button if you want to verify the flash.');
-                    dispatch(warningActions.addWarning('Please physically reset device.'));
+                    logger.error('It seems that the APP does not have ' +
+                        'a Nordic DFU Trigger Interface.');
+                    logger.error('Please physically reset the device by ' +
+                        'clicking the reset button if you want to verify the flash.');
+                    dispatch(warningActions.addUserWarning(
+                        'Seems that Nordic DFU Trigger Interface not found. ' +
```

Comment:
Suggest rewording to something like "It seems that the Nordic DFU Trugger Interface was not found. Please physically reset the device."

Reply:
Sentence has been simplified as "Nordic DFU Trugger Interface was not found. Please physically reset the device."